packages/salesforcedx-utils-vscode/src/auth/authUtil.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Aliases } from '@salesforce/core';
+import { ConfigUtil } from '../config';
+import {
+  DEFAULT_USERNAME_KEY
+} from '../constants';
+import { TelemetryService } from '../telemetry';
+
+export class AuthUtil {
```

Comment:
Minimal auth function necessary to maintain current functioning of workspace context,